ckanext/switzerland/templates/scheming/form_snippets/see_alsos.html

```diff
@@ -24,5 +25,5 @@ <h1 class="hidden-dataset-name">{{ row.data.name }}</h1>
       id="ogdch_see_alsos_add_row"
       class="btn btn-default ogdch-btn-add-row"
       href="#ogdch_see_alsos_pagemark">
-{{ _('Add another related dataset') }}
+{{ _('Add row') }}
```

Comment:
I think I need context to understand how `Add row` is more clear than `Add another related dataset` unless this field does only add a row?